behaviours/dive_control/dive_control/DivingModel.py

```diff
@@ -97,11 +95,11 @@ def __init__(self, node, view, controller, rate=1/10):
         self._error = None
         self._input = None
 
-
-        self._depth_vbs_pid = PIDControl(Kp = 40.0, Ki = 5.0, Kd = 0.0, Kaw = 1.0, u_neutral = 50.0, u_min = 0.0, u_max = 100.0)
-        self._pitch_lcg_pid = PIDControl(Kp = 40.0, Ki = 5.0, Kd = 0.0, Kaw = 1.0, u_neutral = 50.0, u_min = 0.0, u_max = 100.0)
-        self._pitch_tv_pid = PIDControl(Kp = 0.75, Ki = 0.25, Kd = 0.5, Kaw = 1.0, u_neutral = 0.0, u_min = np.deg2rad(-7), u_max = np.deg2rad(7))
-        self._yaw_pid = PIDControl(Kp = 1.0, Ki = 0.0, Kd = 0.0, Kaw = 1.0, u_neutral = 0.0, u_min = np.deg2rad(-7), u_max = np.deg2rad(7))
+        # TODO: Get the parameters from a config file
```

Comment:
Better yet, a ros param where you check if its changed (example in BT) so you can tune it while it runs ;)